articles/api/management/v2/tokens/programatically.md

```diff
@@ -0,0 +1,153 @@
+---
+description: How to programatically generate and use a token for the Management API
+section: apis
+crews: crew-2
+toc: true
+---
+# Programatically generate and use a token for the Management API
```

Comment:
I'd remove the programatically. This is actually the "only" way to properly get a token for the API, the other option is just for test purposes.